Makefile

```diff
@@ -46,7 +46,7 @@ lint: $(VENV)
 	$(BIN)/flake8 talisker tests
 
 _test: $(VENV)
-	. $(BIN)/activate && $(BIN)/pytest --tb=short -n auto $(ARGS)
+	. $(BIN)/activate && $(BIN)/pytest --timeout=15 $(ARGS)
```

Comment:
Huh?

Reply:
Yeah, something is broken with the tests in pytest-xdist. I was regularly getting test hangs in parallel, but not when run normally.Somehow the way I altered the prometheus test made it start happening reliably.

I need to get to the bottom of it, but it's tricky to debug, so I disabled parallel tests for now. I hope to be able to investigate and renable, but I didn't want to block on it.